react-backend/client/src/components/visitors/signup_form.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Input from './input';
+import Select from './select';
+
+export default () => (
+  <div>
+    <section className="Main col-9">
+      <h1>Please tell us about yourself</h1>
+      {error && (
+        <div className="ErrorText">
+          {error.map(el => <span key={el}>{el}</span>)}
+        </div>
+      )}
+      <form className="Signup" onChange={this.handleChange}>
```

Comment:
This wont work, you need to pass down the methods as props. Look up stateless functional components and how to pass props to them.